src/components/text-area/daikin-text-area.ts

```diff
@@ -0,0 +1,204 @@
+
+const cvaTextArea = cva(
+  [
+    "w-full",
+    "h-full",
+    "px-4",
+    "py-3",
+    "rounded-md",
+    "font-daikinSerif",
+    "outline",
+    "outline-[--color-border]",
+    "outline-0",
+    "-outline-offset-2",
+    "placeholder:text-[#616161]",
+
+    // Define `--color-border` as a CSS variable that references `--color-state-focus` and `--color-base` in that order.
+    // `--color-base` indicates the color of the border when the element is normal, hovered, or disabled.
+    "define-[--color-state-active,--color-state-focus,--color-base]/color-border",
+    "border",
+    "border-[--color-border]",
+
+    // Update `--color-base` depending on the state.
+    // The default `--color-base` and `--color-state-focus` values are defined in `variants.error` because they differ depending on whether or not the input has an error state.
+    "enabled:text-[#414141]",
+    "enabled:hover:bg-[#f2f2f2]",
+    "enabled:active:bg-[#EBEBEB]",
+    "focus-visible:outline-2",
+
+    "disabled:var-color-[#BFBFBF]/color-base",
+    "disabled:text-[#BFBFBF]",
+    "disabled:bg-white",
+    "disabled:placeholder:text-[#BFBFBF]",
+  ],
+  {
+    variants: {
+      error: {
+        false: [
+          "enabled:var-color-[#515151]/color-base",
+          "focus-visible:var-color-[#0081C0]/color-state-focus",
+        ],
+        true: ["enabled:var-color-[#D80C18]/color-base"],
+      },
+      resize: {
+        false: ["resize-none"],
+        true: ["resize-vertical"],
+      },
+    },
+  }
+);
+
+/**
+ * The text area component is designed for multiline text input, similar to the HTML `<textarea>` tag.
+ * It is ideal for situations where users need to enter longer pieces of text, such as comments, descriptions, or messages.
+ * Can be used within `daikin-input-group` component.
+ *
+ * Hierarchies:
+ * - `daikin-text-area` (can be used solely)
+ * - `daikin-input-group` > `daikin-text-area`
+ *
+ * @fires change - A cloned event of a [change event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event) emitted from the inner `<textarea>` element.
+ * @fires input - A retargeted event of a [input event](https://developer.mozilla.org/en-US/docs/Web/API/Element/input_event).
+ *
+ * @example
+ *
+ * ```js
+ * import "@daikin-oss/design-system-web-components/components/text-area/index.js";
+ * ```
+ *
+ * ```html
+ * <!-- See `daikin-input-group` component for complete example. -->
+ * <daikin-text-area name="name"></daikin-text-area>
+ * ```
+ */
+@customElement("daikin-text-area")
+export class DaikinTextArea extends LitElement {
+  static override readonly styles = css`
+    ${unsafeCSS(tailwindStyles)}
+
+    :host {
+      display: block;
+      height: 87px;
+      position: relative;
+    }
+  `;
+
+  static readonly formAssociated = true;
+
+  private _internals = this.attachInternals();
+
+  /**
+   * Value of the text area.
+   */
+  @property({ type: String, reflect: true })
+  value = "";
```

Comment:
`value` should not reflect. (same applies to text field.)

```suggestion
  @property({ type: String })
  value = "";
```

Reply:
@poetrainy 
Could you update `daikin-text-field.ts` as well?